File: fixtures/scale/swift/medium/Functions/HyperbolicFunctions.swift
import Foundation

// Hyperbolic trig, registered alongside the ordinary trig functions.
enum HyperbolicFunctions {
    static func install(_ registry: FunctionRegistry) {
        registry.define("sinh") { sinh($0[0]) }
        registry.define("cosh") { cosh($0[0]) }
        registry.define("tanh") { tanh($0[0]) }
    }
}

File: fixtures/scale/java/medium/src/calc/funcs/HyperbolicFunctions.java
package calc.funcs;

import calc.eval.FunctionRegistry;

public class HyperbolicFunctions {
    public static void register(FunctionRegistry registry) {
        registry.addUnary("sinh", Math::sinh);
        registry.addUnary("cosh", Math::cosh);
        registry.addUnary("tanh", Math::tanh);
    }
}
